labgrid/driver/power/netio_json.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Add HTTP-JSON-API functions for labgrid-netio power driver.
+Set model in YML-CFG to 'netio_json'.
+"""
+__author__ = "Eugen Wiens, Christian Happ, Raffael Krakau"
+__copyright__ = "Copyright 2023, JUMO GmbH & Co. KG"
+__email__ = "Eugen.Wiens@jumo.net, Christian.Happ@jumo.net"
+__date__ = "2023-07-05"
+__status__ = "Production"
+
+import json
+
+import requests
+
+PORT = 80
+
+
+class __NetioControl:
```

Comment:
Why the double underscore?

Reply:
Initially dunderscore was added to keep NetioControl private and to prevent it from showing up in the doc.(imo a user/dev does only need this info if he already takes a look in the source-code)

With dunderscore the class will be mangled and show up as power__NetioControl giving a better idea what this exactly is.

Reply:
Can this be resolved?

Reply:
Would a single underscore still be prefered?